Makefile

```diff
@@ -64,7 +65,7 @@ cluster/clean:
 .PHONY: test/unit
 test/unit:
 	@echo Running tests:
-	gotest -v -covermode=count -coverprofile=coverage.out ./pkg/...
```

Comment:
Can you add this back again :)

Reply:
We opted to go back to `go test` to avoid adding more dependencies to the repo